index.js

```diff
@@ -31,29 +31,28 @@ ghauth(authOptions, function (err, authData) {
       const assets = []
       res.data.forEach(branch => {
         platforms.forEach(platform => {
-          assets.push({
-            branch: branch.name,
-            commit: branch.commit.sha,
-            platform: platform,
-            url: `${url}/${platform}/${branch.commit.sha}/libchromiumcontent.zip`
-          })
-          assets.push({
-            branch: branch.name,
-            commit: branch.commit.sha,
-            platform: platform,
-            url: `${url}/${platform}/${branch.commit.sha}/libchromiumcontent-static.zip`,
-            static: true
-          })
+          if (!query || query === branch.commit.sha) {
```

Comment:
This if-statement could be placed earlier, right? Rather than in this loop...

Reply:
Yup, you are right.  I'll correct it.